apps/web/src/app/(dashboard)/documents/page.tsx

```diff
@@ -61,7 +61,7 @@ export default async function DocumentsPage({ searchParams = {} }: DocumentsPage
 
   return (
     <div className="mx-auto w-full max-w-screen-xl px-4 md:px-8">
-      <UploadDocument />
+      <UploadDocument verifiedUser={!!user.emailVerified} />
```

Comment:
This could probably be retrieved from the `useSession` hook with the right tweak to our jwt function in `auth-options.ts`

Reply:
Really good suggestion! I didn't think of this.